infrastructure/compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+name: planthor-webapi-infrastructure
+
+services:
+
+  mongodb:
+    image: mongo:7
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=admin
+      - MONGO_INITDB_ROOT_PASSWORD=planthor_123
```

Comment:
do we need to set the password the same as IDP?

Reply:
2 things:
- I will update with Upper P: --> `Planthor_123`
- But for underscore `_`, I cannot change to `@`, as it could confuse with connection string of Mongo db `<username>@<password>`

Reply:
updated.